todo_app/data/trello_request_handler.py

```diff
@@ -0,0 +1,145 @@
+from abc import ABC, abstractmethod
+import json
+import requests
+
+class trello_request_handler(ABC):        
+    @abstractmethod 
+    def get_board(self):
+        pass
+       
+    @abstractmethod 
+    def get_all_cards(self):
+        pass
+       
+    @abstractmethod 
+    def get_all_lists(self):
+        pass
+
+    @abstractmethod 
+    def update_list_on_card(self, cardId, listId):
+        pass
+
+    @abstractmethod 
+    def add_new_card(self, cardName, listId):
+        pass
+
+    @abstractmethod
+    def delete_card(self, cardId):
+        pass
+
+class fake_trelllo_request_handler(trello_request_handler):   
```

Comment:
Why did you define this?

Reply:
this was so that I could test my app without hitting the real trello website every time